mks/model/stuf_02_04.py

```diff
@@ -441,6 +443,17 @@ def set_omschrijving_geslachtsaanduiding(target):
     target['omschrijvingGeslachtsaanduiding'] = geslacht
 
 
+def set_reden_ontbinding_omschrijving_custom(verbintenis, reden):
+    reden_custom = None
+
+    if reden == '1':
```

Comment:
reden ontbinding is een string!
Zie tabel 41
https://publicaties.rvig.nl/Landelijke_tabellen/Landelijke_tabellen_32_t_m_60_excl_tabel_35/Landelijke_Tabellen_32_t_m_60_in_csv_formaat?pagenr=2

Reply:
'1' is toch een string?

Reply:
Ik zal even de correcte code neerzetten.